cmp/grid/Grid.ts

```diff
@@ -108,13 +110,35 @@ export const [Grid, grid] = hoistCmp.withFactory<GridProps>({
         );
 
         return fragment(
-            frame({
+            vframe({
                 className,
-                item: agGrid({
-                    model: model.agGridModel,
-                    ...getLayoutProps(props),
-                    ...impl.agOptions
-                }),
+                items: [
+                    agGrid({
+                        model: model.agGridModel,
+                        ...getLayoutProps(props),
+                        ...impl.agOptions
+                    }),
+                    div({
```

Comment:
I would make this its own HoistComponent to avoid extra re-rendering whenever the scrollbar calcs need to change.  ALSO, weird that the (experimental) scrollbar dominates the render method the way it does now.

Ideally put *all* the state for this in a dedicated GridScrollBarModel/GridScrollBar pair in their own files.

Reply:
Especially nice to have these split out while this is experimental, but also will be useful even if this becomes built-in